src/System.Management.Automation/engine/CommandCompletion/CommandCompletion.cs

```diff
@@ -591,6 +591,8 @@ private static CommandCompletion CompleteInputImpl(Ast ast, Token[] tokens, IScr
 
                     var completionResults = results ?? EmptyCompletionResult;
                     sw.Stop();
```

Comment:
Same here, shall we remove the timer code?

Reply:
fixed